README.md

```diff
@@ -4,81 +4,74 @@
   ![Master branch unit testing result](https://travis-ci.org/hackcambridge/hack-cambridge-website.svg?branch=master)
 ](https://travis-ci.org/hackcambridge/hack-cambridge-website)
 
-## Getting Started
+## Getting started
 
-To run the website on your machine, make sure you have [Node.js](https://nodejs.org) and [Yarn](https://yarnpkg.com/) installed. Then inside this folder, run
+To run the website on your machine, first make sure you have [Node.js](https://nodejs.org), [Yarn](https://yarnpkg.com/), and [Docker](https://www.docker.com) installed.  Then [clone this repository from GitHub](https://help.github.com/articles/cloning-a-repository/):
 
-```bash
```

Comment:
What was the reason for removing this (especially given that it remains in [line 68](https://github.com/hackcambridge/hack-cambridge-website/pull/509/files#diff-04c6e90faac2675aa89e2176d2eec7d8R68))?

Reply:
`gulp serve` is the last step so I moved it into that order.

Reply:
No I mean removing `bash`—that's used for [syntax highlighting](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown)

Reply:
Ah I see, yes that should have stayed, thanks for the spot!

Reply:
And it carries through to basically all the other code blocks